requirements-dev.txt

```diff
@@ -13,4 +13,5 @@ sphinx==7.1.*
 nbsphinx
 sphinx_rtd_theme
 
+idaes-pse==2.10.0rc0
```

Comment:
Is there a specific reason why this is repeated in both `requirements-dev.txt` and `setup.py`? If not, I'd suggest to remove this line so that only `setup.py` remains.

```suggestion
```